UpcomingMovies/UpcomingMovies/Services/RemoteService.cs

```diff
@@ -0,0 +1,195 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UpcomingMovies.Models;
+using UpcomingMovies.Models.Json;
+using UpcomingMovies.Settings;
+using Xamarin.Forms;
+using System.Net.Http;
+using Newtonsoft.Json;
+
+[assembly: Dependency(typeof(UpcomingMovies.Services.RemoteService))]
+namespace UpcomingMovies.Services
+{
+    class RemoteService : IRemoteService<IMovieItem>
+    {
+        public List<IMovieItem> Items { get; private set; }
+        public IRemoteSettings Settings { get; private set; }
+
+        public static Dictionary<int, string> GenresDict = new Dictionary<int, string>()
+        {
+            { 28, "Action" },
+            { 12, "Adventure" },
+            { 16, "Animation" },
+            { 35, "Comedy" },
+            { 80, "Crime" },
+            { 99, "Documentary" },
+            { 18, "Drama" },
+            { 10751, "Family" },
+            { 14, "Fantasy" },
+            { 10769, "Foreign" },
+            { 36, "History" },
+            { 27, "Horror" },
+            { 10402, "Music" },
+            { 9648, "Mystery" },
+            { 10749, "Romance" },
+            { 878, "Science Fiction" },
+            { 10770, "TV Movie" },
+            { 53, "Thriller" },
+            { 10752, "War" },
+            { 37, "Western" },
```

Comment:
It is available in the API, https://api.themoviedb.org/3/genre/movie/list?api_key=1f54bd990f1cdfb230adb312546d765d&language=en-US. 

In the future implementations it would be dinamically updated using the above link.